vertices.go

```diff
@@ -43,6 +43,33 @@ func (v *verticesBackend) get() []float32 {
 	return s
 }
 
+// compute lowest power of two equal to or greater than n.
```

Comment:
Start with `npo2 is` for Go convention

BTW, I'd use `internal/math.NextPowerOf2Int`. I believe performance is not so different. Have you compared this?

Reply:
I would expect it to be noticably worse, in most cases. Branches are horrendously expensive, and loops that will usually loop several times are going to be *way* more expensive than a small number of operations. Not that this is a very expensive part of the rendering as a whole, but for the actual cost of this operation, I'd expect probably around an order of magnitude difference. The bit operation thing ends up being about 12 instructions, and can be inlined (at least in non-desktop go) because there's no loops. The loop version can't be inlined, and while it might be a couple fewer instructions, for even a small image (say, 64px), it's going to be looping at least six times.

Reply:
As there is a benchmark `BenchmarkDrawImage` in image_test.go, let's compare the performance. I admit theoretically it is possible that your code is faster, but this seems a too-early optimization to me.

Reply:
On second thought, as the current code doesn't use `NextPowerOf2Int` anyway, I'm fine with your npo2. Thank you for discussing at Slack.

Please fix the comment anyway.

Reply:
I started to think about the performance of `npo2` on browsers: on browsers, a lot of range checks are inserted in interger operations. I don't think this is not too much in this case, but am not sure.